src/main/resources/application-lrd-building-location-router.yaml

```diff
@@ -11,9 +11,9 @@ route:
       sql:INSERT INTO building_location (epimms_id, building_location_name, building_location_status, area, region_id, cluster_id, court_finder_url, postcode, address, welsh_building_location_name, welsh_address, uprn, latitude, longitude, mrd_building_location_id,mrd_created_time,mrd_updated_time,mrd_deleted_time, created_time, updated_time)
       VALUES (:#epimms_id,:#building_location_name,:#building_location_status,:#area,:#region_id,:#cluster_id,:#court_finder_url,:#postcode,:#address,:#welsh_building_location_name,:#welsh_address,:#uprn,:#latitude,:#longitude,:#mrd_building_location_id,:#mrd_created_time,:#mrd_updated_time,:#mrd_deleted_time,NOW() AT TIME ZONE 'utc',NOW() AT TIME ZONE 'utc')
       ON CONFLICT (epimms_id) DO UPDATE SET building_location_name = :#building_location_name,building_location_status = :#building_location_status,area = :#area,region_id = :#region_id,cluster_id = :#cluster_id,
-      court_finder_url = :#court_finder_url,postcode = :#postcode,address = :#address,welsh_building_location_name = :#welsh_building_location_name,welsh_address = :#welsh_address,uprn = :#uprn,latitude = :#latitude,longitude = :#longitude,mrd_building_location_id = :#mrd_building_location_id,mrd_created_time = :#mrd_created_time,mrd_updated_time = :#mrd_updated_time,mrd_deleted_time = :#mrd_deleted_time,updated_time = NOW() AT TIME ZONE 'utc'?batch=true&dataSource=dataSource
+      court_finder_url = :#court_finder_url,postcode = :#postcode,address = :#address,welsh_building_location_name = :#welsh_building_location_name,welsh_address = :#welsh_address,uprn = :#uprn,latitude = :#latitude,longitude = :#longitude,mrd_building_location_id = :#mrd_building_location_id,mrd_created_time = :#mrd_created_time,mrd_updated_time = :#mrd_updated_time,mrd_deleted_time = :#mrd_deleted_time,updated_time = NOW() AT TIME ZONE 'utc'?dataSource=#dataSource
```

Comment:
`batch=true` no longer works with the query written as is - if required may have to rewrite.
`dataSource` requires the `#` identifier now for beans.